crafting_tweaks/files/quality_of_life/back_to_blocks/recipes/back_to_blocks/slabs/smooth_sandstone.recipe.json

```diff
@@ -14,19 +14,16 @@
 		],
 		"key": {
 			"#": {
-				"item": "minecraft:stone_block_slab2",
-				"data": 6
+				"item": "minecraft:smooth_sandstone"
 			}
 		},
 		"unlock": [
 			{
-				"item": "minecraft:stone_block_slab2",
-				"data": 6
+				"item": "minecraft:smooth_sandstone"
```

Comment:
Should be `"minecraft:smooth_sandstone_slab"` not `"minecraft:smooth_sandstone"` lol